docker-compose.yaml

```diff
@@ -2,9 +2,14 @@ services:
   ffc-pay-statement-receiver:
     build:
       context: .
-      target: production
+      target: development
```

Comment:
This should be production so build in pipeline targets production Dockerfile stage

Reply:
If this is production does in config `value.env` not become `production` which causes the `isProd` to be true which causes Redis to attempt to load with TLS which breaks it?